### ISSUE TITLE
Fody always rebuilds even if no change in the source has occured.

### ISSUE THREAD

**Issue by bradphelan** [Fody/Fody#410]
The output of my project build is always

========== Build: 30 succeeded, 0 failed, 1 up-to-date, 0 skipped ==========

every time I press build. The build for each sub project is always similar to

```
26>  Fody: Fody (version 2.1.3.0) Executing
26>  Fody:   Finished Fody 8ms.
27>  Fody: Fody (version 2.1.3.0) Executing
27>  Fody:   Finished Fody 9ms.
27>WeinCad -> C:\Users\phelan\workspace\WeinCAD.NET\WeinCad.Net\bin\Debug\net461\win7\WeinCad.exe
27>  Fody:   Skipped Verifying assembly since it is disabled in configuration
27>  Fody:   Finished verification in 3ms.
27>Done building project "WeinCad.csproj".
```

**Comment by bradphelan** [comment]
Seems to be a regression on https://github.com/Fody/Fody/issues/167

**Comment by SimonCropp** [comment]
Has your research discovered any fixes

**Comment by bradphelan** [comment]
My research? To me Fody is a box of magic fairy dust I sprinkle on my
project to make stuff happen. I wouldnt know where to start to find a fix.
I was hoping a wizard or two is around and explain what's up. Maybe I've
applied the fairy dust incorrectly. I don't really know.

I just know that I'm fairly sure at some point a while agonit was working
and then I noticed it was not working. It could be when I upgraded to using
new SDK format csproj files. I'll try and setup a simple test when I get
into work to see if that is true.
